test/server/graphql/v2/query/ApplicationQuery.test.ts

```diff
@@ -16,6 +16,7 @@ const applicationQuery = gqlV2/* GraphQL */ `
       redirectUri
       clientId
       clientSecret
+      lastUsedAt
```

Comment:
`lastUsedAt` lives in `OAuthAuthorization`, not `Application` (this is the reason why tests are failing)

Reply:
Thanks for the note, I already fixed it.

However, one of the tests is still failing and the reason is that the test data is not correctly instantiated. I tried to create the `OAuthAuthorization`, but I was not successful. Can you help me at this point?

Reply:
Thanks, I'll take it from here